scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +65,21 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val, metric_type):
+        """
+        :param val: string representing metric value
+        :param metric_type: string or numeric
+        :return: value converted to float if metric_type isn't string
```

Comment:
This comment is no longer valid. It'll convert it to float or to int depending on parsing.